tests/testthat/test_get_var_y.R

```diff
@@ -111,7 +111,7 @@ test_that("Two compartment model", {
     parameters = par2,
     t_obs = t_obs,
     regimen = reg,
-    mega = omega2
+    omega = omega2
```

Comment:
actually because of the ellipsis this wasn't caught before. The code ran, but the omega argument wasn't specified so the computations were wrong. We just had ridiculously high tolerance on the test values, so we didn't see it.